sql/updates/world/2016_01_25_09_world.sql

```diff
@@ -0,0 +1,56 @@
+(4488,3,0,"Ah, Ner'zhul. Not a name that should be uttered lightly, but I did ask you for an answer.$B$BAnd Ner'zhul is the correct one.",              12,0,100,1,3000,0,8376,0,'Parqual replies to option 4'),
+(4488,4,0,"I'm afraid not, $n. Now, feel the shame you have brought upon yourself.",                                                                    12,0,100,1,3000,0,8394,0,'Parqual reply 2 to option 1-3');
+
+-- new SmartAI script for NPC 4488 Parqual Fintallas:
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 4488 AND `source_type` = 0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(4488, 0,  0,  1, 62, 0, 100, 0, 4764,    0, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 0 selected - Close Gossip"),
+(4488, 0,  1,  0, 61, 0, 100, 0,    0,    0, 0, 0,  1,    0, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 0 selected - Say Text 0"),
+(4488, 0,  2,  0, 52, 0, 100, 0,    0, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 0 Over - Say Text 4"),
+(4488, 0,  3,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
+(4488, 0,  4,  5, 62, 0, 100, 0, 4764,    1, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 1 selected - Close Gossip"),
+(4488, 0,  5,  0, 61, 0, 100, 0,    0,    0, 0, 0,  1,    1, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 1 selected - Say Text 1"),
+(4488, 0,  6,  0, 52, 0, 100, 0,    1, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 1 Over - Say Text 4"),
+(4488, 0,  7,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
+(4488, 0,  8,  9, 62, 0, 100, 0, 4764,    2, 0, 0, 72,    0,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 2 selected - Close Gossip"),
+(4488, 0,  9,  0, 61, 0, 100, 0,    0,    0, 0, 0,  1,    2, 5000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Gossip Option 2 selected - Say Text 2"),
+(4488, 0, 10,  0, 52, 0, 100, 0,    2, 4488, 0, 0,  1,    4, 3000, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 2 Over - Say Text 4"),
+(4488, 0, 11,  0, 52, 0, 100, 0,    4, 4488, 0, 0, 11, 6767,    0, 0,0,0,0, 7, 0,0,0,0,0,0,0,"Parqual Fintallas - on Text 4 Over - Cast spell 6767 'Mark of Shame'"),
```

Comment:
OK. I guess I tried using SMART_ACTION_LINK at first, but when that didn't work, I forgot to consider simplification.
Fixing this in a few minutes.